js/5etools-backgrounds.js

```diff
@@ -115,16 +117,49 @@ function d20plusBackgrounds () {
 
 		const renderer = new Renderer();
 		renderer.setBaseUrl(BASE_SITE_URL);
-		const renderStack = [];
-		let feature = {};
+		const featureSet = [
+			"Cultural Chameleon",
+			"Dust Digger",
+			"Favored Event",
+			"Specialty",
+			"Contacts",
+			"How Do I Fit In?",
+			"Favorite Schemes",
+			"Faceless Persona",
+			"Why Are You Here?",
+			"Fey Mark",
+			"Feywild Visitor",
+			"Fishing Tale",
+			"Harrowing Event",
+			"Role",
+			"Path to Mystery",
+			"Hardship Endured",
+			"Variant Noble (Knight)",
+			"A Flair for the Dramatic",
+			"Life at Sea",
+			"Claim to Fame",
+			"Carnival Companion"
+		];
+		let features = [];
 		bg.entries.forEach(e => {
+			let feature = {};
 			if (e.name && e.name.includes("Feature:")) {
 				feature = JSON.parse(JSON.stringify(e));
```

Comment:
`MiscUtil.copy( ... )` is available instead of `JSON.parse(JSON.stringify( ... ))`, fwiw

Reply:
Switched code to use MiscUtil.copy()